Doc/library/enum.rst

```diff
@@ -52,11 +52,11 @@ are not normal Python classes.  See
 
 .. note:: Nomenclature
 
-   - The class :class:`Color` is an *enumeration* (or *enum*)
-   - The attributes :attr:`Color.RED`, :attr:`Color.GREEN`, etc., are
+   - The class :class:`!Color` (or *enum*)
```

Comment:
```suggestion
   - The class :class:`!Color` is an *enumeration* (or *enum*)
```

``is an *enumeration*`` was evidently accidentally dropped from the original text in the latest revision, which made it no longer make grammatical or semantic sense.

FYI, applying the given review suggestions (with [`Files changed`](https://github.com/python/cpython/pull/101122/files) -> <kbd>Add to batch</kbd> -> <kbd>Commit</kbd>) automatically instead of trying to duplicate them manually in your local repo helps avoid these sort of mistakes (or at least means, if they happen, they're 100% my fault :laughing: ). To note, you can revise the suggestions given by making a comment of your own, copying the suggestion in and modifying it to your liking.